src/main/java/seedu/address/model/person/Supplier.java

```diff
@@ -119,20 +119,6 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        // TODO Change the string representation
```

Comment:
Do we just use the default Person.toString() method? I think this depends on the UI implementation?

Reply:
Currently there are no methods calling this toString() method, hence I feel that it is fine to do so for now